### ISSUE TITLE
FogChamber static scenes labels

### ISSUE THREAD

**Issue by motapinto** [rheinzler/PointCloudDeNoising#23]
In the paper it mentions that 4 realistic static setups are represented in the data from the climate chamber. I presume that the names of the folder that contain Static1, Static2, Static3, Static4 refer to this, however I noted that the train_01 folder only contains Static1 data, train_02 Static4, the test_01 folder only contains Static2 and the val_01 Static3. Is there any reasons for this? Shouldn't the training data contain data from all 4 static setups?

**Comment by rheinzler** [comment]
Of course there is a reason for this. The data is carefully split to prevent the CNN from overfitting or memorize the scene. Each static scene contains time correlated data i.e. frame t_1 and t_2 might look identical. As we recorded static scenes this even holds for any frame for one scene. 

This is also described in the paper. Please have a look at section IV C
